src/lib/vina.cpp

```diff
@@ -716,17 +717,18 @@ std::vector<double> Vina::score(double intramolecular_energy) {
 	if (m_sf_choice == SF_VINA || m_sf_choice == SF_VINARDO) {
 		// Inter
 		if (m_no_refine || !m_receptor_initialized)
-			lig_grids = m_grid.eval(m_model, authentic_v[1]); // [1] ligand - grid
+			all_grids = m_grid.eval(m_model, authentic_v[1]); // [1] ligand + flex - grid
```

Comment:
Minor suggestion: Instead of `ligand + flex - grid` (which at first glance looks like addition and subtraction (i know it doesn't make too much sense in context but a code comment shouldn't require too much context)) maybe we could use something a bit more descriptive like `ligand & flex -- grid eval`.

Reply:
Agreed, thanks1